internal/taskfile/taskfile.go

```diff
@@ -40,3 +40,61 @@ func (tf *Taskfile) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	}
 	return nil
 }
+
+func (t *Task) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var task struct {
```

Comment:
Hey @jaedle,

IMHO you don't need to copy the entire task struct into this function. Just try to marshal the `string` and `[]string` types first, and none work then unmarshal to the `t` variable directly. 🙂

Reply:
Oops, I just remembered that we also have alternative command syntaxes, like `{ cmd: "echo foo", silent: true }` for example.

That's easy to do, though. Just check for `taskfile.Cmd` and `[]taskfile.Cmd` instead of for `string` and `[]string`. All variations should just work.